azkaban-exec-server/src/main/java/azkaban/execapp/FlowPreparer.java

```diff
@@ -47,6 +43,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static azkaban.utils.ThinArchiveUtils.*;
```

Comment:
Looks like the the * imports got introduced by IDE or by formatting. Might want to change these to individual imports.

Reply:
This file change should not have been committed. I have reverted it.